### ISSUE TITLE
Need to include `using LinearAlgebra` for `svd` to work

### ISSUE THREAD

**Issue by gvdeynde** [JuliaLinearAlgebra/GenericLinearAlgebra.jl#89]
```
using GenericLinearAlgebra
svd(big.([1,2;3,4]))
```
returns
```
UndefVarError: svd not defined

Stacktrace:
 [1] top-level scope
   @ In[2]:1
 [2] eval
   @ .\boot.jl:373 [inlined]
 [3] include_string(mapexpr::typeof(REPL.softscope), mod::Module, code::String, filename::String)
   @ Base .\loading.jl:1196
```

If I add `using LinearAlgebra` before `using GenericLinearAlgebra`, it works. 

This is, to me, a bit counter-intuitive.

**Comment by andreasnoack** [comment]
Yeah. It would make sense to reexport `LinearAlgebra`. It's unlikely that you'd only like to load this package without `LinearAlgebra.

**Comment by LilithHafner** [comment]
The `svd` function is defined in `LinearAlgebra`, so unfortunately with the current design you will need LinearAlgebra for that. On the other hand, unless you are manually excising packages from the sysimg you need LinearAlgebra for `println("hello world")`.

Tangentially related: https://github.com/JuliaLang/julia/issues/48098
